public-images/public-images-explanation/lxd-openstack-images.rst

```diff
@@ -0,0 +1,210 @@
+handle virtualisation itself, but acts as a wrapper that leverages
+existing virtualisation technologies.
+
+
+What are these images?
+----------------------
+
+Canonical provides cloud image artefacts on
+`cloud-images.ubuntu.com <https://cloud-images.ubuntu.com/>`__ that have
+been customised to run on public clouds, including LXD and OpenStack. To
+learn more about these artefacts and supported architectures, visit our
+Ubuntu cloud image artefacts documentation :doc:`here <../public-images-reference/artefacts>`.
+
+
+How do you access them?
+-----------------------
+
+Go to `cloud-images.ubuntu.com <https://cloud-images.ubuntu.com/>`__ and
+select a release. For the latest LTS release, Ubuntu 24.04 LTS Noble
+Numbat, you would navigate to ``noble > current``. Note that all artefacts are
+architecture specific, in the format
+``<release>-<type>-cloudimg-<architecture>-<artefact>``.
+
+LXD and OpenStack also have `Minimal Cloud
+Images <https://cloud-images.ubuntu.com/minimal/>`__:
+Ubuntu images that have a reduced runtime footprint, optimised kernel
+and boot process. They are smaller and boot faster, but are not designed
+for environments requiring human interaction or debugging.
+
+
+LXD images
+~~~~~~~~~~
+
+To import an image into LXD, you will need two artefacts:
+
+1. An LXD tarball:
+
+   - The :ref:`lxd-tarball-ref` artefact has the extension ``*.lxd.tar.xz``
+   - It contains the metadata needed by LXD to instantiate a container or virtual machine as well as a folder for any custom templates
+
+2. A file system for a container or a bootable disk image for a virtual
+   machine:
+
+   - The file system for a container can be either a 
+
+     * :ref:`root-tarball-ref` (``*-root.tar.xz``), or a 
+
+     * :ref:`squashfs-ref` (``*.squashfs``)
+
+   - The bootable disk image for a virtual machine is a :ref:`qcow-ref` image (``*.img``)
+
+The following are example commands to import LXD containers and virtual machines, respectively, based on downloaded Ubuntu 24.04 artefacts:
+
+.. code :: bash
+
+   lxc image import noble-server-cloudimg-amd64-lxd.tar.xz \
+      noble-server-cloudimg-amd64-root.tar.xz --alias noble_container
+
+.. code :: bash
+
+   lxc image import noble-server-cloudimg-amd64-lxd.tar.xz \
+      noble-server-cloudimg-amd64.img --alias noble_vm
+
+
+OpenStack images
+~~~~~~~~~~~~~~~~
+
+OpenStack uses QCOW images. Download the artefact for your chosen
+architecture with the ``*.img`` extension.
+
+Use the `OpenStack command-line
+client <https://docs.openstack.org/ocata/admin-guide/common/cli-install-openstack-command-line-clients.html>`__
+to interact with OpenStack. An example of uploading an image looks 
+something like this:
+
+.. code :: bash
+
+   openstack image create “Ubuntu-24.04” \
+      --file noble-server-cloudimg-amd64.img \
+      --disk-format qcow2 \
+      --container-format bare \
+      --public
+
+To learn more about managing images with OpenStack, you can read their
```

Comment:
```suggestion
To learn more about managing images with OpenStack, you can refer to the `Manage images`_ section of their
```
We try to avoid using 'here' for link descriptions and instead use the expected content title as is. Since the whole page seems relevant I'm using the page title (& a link to it). If you think the specific section that you pointed to makes more sense, we can use that too.

Reply:
Good to know, `here` felt awkward.